src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue

```diff
@@ -74,6 +94,14 @@ export default {
 		selectedChannelLoanIds() {
 			return this.selectedChannel?.loans?.values?.map(loan => loan.id) ?? [];
 		},
+		name() {
```

Comment:
We can use these to pass is the name/description to show them instantly if the parent component knows them while this component does it's loading